Collections refactor (#21)
kfoley123/pokemoncard_frontend: README.md

# _Pokemon Card Database_

> A database of Pokemon Cards that you can view or add to personal collections.

Reviewer: this app is not a database, but it does interact with one. You may want to specificy that this is a frontend that connects to an api of Pokemon Cards (which you can link to that repo) that lets you view or add cards to a personal collection

---

## Authors

Kortney Foley

---

## Technologies Used

-   HTML
-   JavaScript
-   CSS
-   React
-   [React Router](https://reactrouter.com/en/main)

---

## Description

The purpose of designing this app was to develop my back end and database creation skills. Original version of the app had full CRUD functionality so card could be updated, added or deleted, but it didn't make sense to keep all these functions for general users, so most of these have been removed or limited to "admin" account user.

The App alows users to view all Pokemon cards that have been added to the database, sort them by different filters (i.e type and/or specifc card set) and create a user if signed out. When signed in, users can create their own collections of cards.

TODO:

-   update CSS to work on mobile

> -   optomize for accessibility

Reviewer: ```suggestion
-   optimize for accessibility
```

# _Pokemon Card Database_

> A frontend app that links to an API of Pokemon Cards that you can view or add to personal collections.

Reviewer: ```suggestion
A frontend app that links to an API of Pokemon Cards which allows users to create an account, log in to their account and add or remove cards from their personal collection
```

maybe something like that?

---

## Authors

Kortney Foley

---

## Technologies Used

-   HTML
-   JavaScript
-   CSS
-   React
-   [Axios](https://www.npmjs.com/package/axios)

> -   [React Query](https://tanstack.com/query/v4/?from=reactQueryV3&original=https://react-query-v3.tanstack.com/)

Reviewer: ```suggestion
-   [React Query](https://tanstack.com/query/v4/)
```

-   [React Router](https://reactrouter.com/en/main)

---

## Description

> The purpose of designing this app was to develop my back end and database creation skills. Original version of the app had full CRUD functionality so card could be updated, added or deleted, but it didn't make sense to keep all these functions for general users, so most of these have been removed or limited to "admin" account user.

Reviewer: ```suggestion
The purpose of designing this app was to develop my back end and database creation skills. Original version of the app had full CRUD functionality so that a card could be updated, added or deleted, but it didn't make sense to keep all these functions for general users, so most of these have been removed or limited to "admin" account user.
```
